tests/unit/test_databricks.py

```diff
@@ -209,6 +209,29 @@ def test_create_table_replace(databricks):
     )
     assert replaced is True
 
+    databricks.drop_table(schema=TEST_SCHEMA, table=TEST_TABLE)
+    databricks.drop_schema(TEST_SCHEMA)
+
+
+def test_replace_not_the_same_column_schema(databricks):
```

Comment:
I would replace "not_the_same" with "different"

Reply:
[✅ Updated name and comment i databricks tests](https://github.com/dyvenia/viadot/pull/658/commits/dc279bf072230139794efe2ac3719ed64f972316)